### PR TITLE
Fixes #7542: Tests should run ncf in \"debug\" mode to have more output to make it easier to analyse test failures

### PR DESCRIPTION
**Pull request by jooooooon**
https://www.rudder-project.org/redmine/issues/7542

### CODE REVIEW

**Comment by jooooooon** [comment]
PR updated

**Comment by jooooooon** [comment]
PR updated

File: tests/acceptance/testall [hunk 1]
```diff
@@ -247,7 +247,7 @@ export CFENGINE_TEST_OVERRIDE_WORKDIR TEMP NCF_TREE NCF_TESTS_ACCEPTANCE
       printf "gdb --args " >> "$WORKDIR/runtest"
     fi
 
-    printf "\"$AGENT\" $VERBOSE -Kf \"$TEST\" -D AUTO,DEBUG 2>&1\n" >> "$WORKDIR/runtest"
+    printf "\"$AGENT\" $VERBOSE -Kf \"$TEST\" -D AUTO,DEBUG2>&1\n" >> "$WORKDIR/runtest"
```
Comment (peckpeck) [inline tests/acceptance/testall]: There's a missing space here

**Comment by jooooooon** [comment]
PR updated

**Comment by jooooooon** [comment]
PR updated

**Comment by peckpeck** [comment]
Good, merging

**Pull request merged**